### PR TITLE
Fixing validation on create repository dialog

### PR DESCRIPTION
**Pull request by StanleyGoldman**
Related #2264 

Depends on:
- [x] #2252 being merged to **master**

image

image

image

image

### CODE REVIEW

File: src/GitHub.VisualStudio.UI/Views/Dialog/RepositoryCreationView.xaml [hunk 1]
```diff
@@ -107,17 +107,21 @@
                           Grid.Column="1"
                           MaxLength="{x:Static ghfvs:Constants.MaxRepositoryNameLength}"
                           Text="{Binding RepositoryName, UpdateSourceTrigger=PropertyChanged}" 
-                          SpellCheck.IsEnabled="True"
```
Comment (jcansdale) [inline src/GitHub.VisualStudio.UI/Views/Dialog/RepositoryCreationView.xaml]: It looks like spell checking has been disabled. Was this by design?

Comment (StanleyGoldman) [reply src/GitHub.VisualStudio.UI/Views/Dialog/RepositoryCreationView.xaml]: Yea, I disabled it, when I couldn't type `VisualStudio` without it calling that an error, I decided that it was not useful...

**Comment by StanleyGoldman** [comment]
Added screenshots

File: src/GitHub.VisualStudio.UI/Views/Dialog/RepositoryCreationView.xaml (hunk 1 above)
Comment (jcansdale) [reply src/GitHub.VisualStudio.UI/Views/Dialog/RepositoryCreationView.xaml]: Lol, makes sense. Though I'm perfectly capable of misspelling the name of my repo. 😉

Comment (jcansdale) [reply src/GitHub.VisualStudio.UI/Views/Dialog/RepositoryCreationView.xaml]: It look like dotcom also has spell checking disabled fro this field.

image

Comment (Loki777m) [reply src/GitHub.VisualStudio.UI/Views/Dialog/RepositoryCreationView.xaml]: ok

**Pull request merged**